src/components/ArticlesPage.js

```diff
@@ -15,13 +16,35 @@ const styles = StyleSheet.create({
 });
 
 export default class ArticlesPage extends Component { // eslint-disable-line
+  constructor(props) {
+    super(props)
+
+    const ds = new ListView.DataSource({
+      rowHasChanged: (r1, r2) => r1 !== r2,
```

Comment:
This is a required parameter that should tell if rows are different and need re-rendering.